instrumentation/src/javatest/org/oppia/android/instrumentation/ExplorationPlayerTest.kt

```diff
@@ -0,0 +1,108 @@
+package org.oppia.android.instrumentation
+
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.UiCollection
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiScrollable
+import androidx.test.uiautomator.UiSelector
+import androidx.test.uiautomator.Until.hasObject
+import org.junit.Assert.assertNotNull
+import org.junit.Before
+import org.junit.Test
+
+/** Tests for Explorations. */
+class ExplorationPlayerTest {
```

Comment:
I suggest also adding another basic test that just verifies that the app starts up successfully. While not leveraging the full end-to-end capabilities that this test eventually will, it's a good baseline test to ensure the UiAutomator environment is working correctly without the additional complexity that this test will require to also be working.

Reply:
@BenHenning can you specify the test suite and the test case particularly,

In meeting we decided to add a exploration player test itself

Reply:
Ah, I just mean verifying that the SplashActivity starts up. I don't think anymore more complicated than that is needed.

Reply:
I wasn't able to test the splash screen, 
It was too quick to capture in UiAutomatorViewer and I couldn't catch any views in it.

Reply:
Got it. In that case, let's proceed with your current test & the later changes you're planning. Thanks for trying.